.github/workflows/release.yml

```
@@ -392,9 +392,17 @@ jobs:
       - run: |
           dart pub global activate melos 6.3.2
           melos bootstrap
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)